src/tnglib/monitor.py

```diff
@@ -160,3 +160,35 @@ def get_metrics(vnf_uuid, vdu_uuid):
         LOG.debug("Request returned with " + (json.dumps(templates)))
         error = "VDUs not found"
         return False, error
+
+def get_metric(metric_name):
+    """
+    Returns value per metric name.
+
+    """
+    resp = requests.get(env.monitor_api+'/prometheus/metrics/name/'+metric_name,
```

Comment:
Hi Panos. We are trying to be conform pep8 python standard, so line lengths can't exceed 80 chars. Can you split this one up please?